frontend/src/scenes/experiments/ExperimentView/components.tsx

```diff
@@ -335,3 +335,152 @@ export function PageHeaderCustom(): JSX.Element {
         />
     )
 }
+
+export function ActionBanner(): JSX.Element {
+    const {
+        experiment,
+        experimentInsightType,
+        experimentResults,
+        experimentLoading,
+        experimentResultsLoading,
+        isExperimentRunning,
+        areResultsSignificant,
+        isExperimentStopped,
+        funnelResultsPersonsTotal,
+        recommendedSampleSize,
+        actualRunningTime,
+        recommendedRunningTime,
+        getHighestProbabilityVariant,
+    } = useValues(experimentLogic)
+
+    if (!experiment || experimentLoading || experimentResultsLoading) {
+        return <></>
+    }
+
+    // Draft
+    if (!isExperimentRunning) {
+        return (
+            <LemonBanner type="info" className="mt-4">
+                Your experiment is in draft mode. You can edit your variants, adjust release conditions, and{' '}
+                <Link className="font-semibold" to="https://posthog.com/docs/experiments/testing-and-launching">
+                    test your feature flag
+                </Link>
+                . Once everything works as expected, you can launch your experiment. From that point, any new experiment
+                events will be counted towards the results.
+            </LemonBanner>
+        )
+    }
+
+    // Running, results present, not significant
+    if (isExperimentRunning && experimentResults && !isExperimentStopped && !areResultsSignificant) {
+        // Results insignificant, but a large enough sample/running time has been achieved
+        // Further collection unlikely to change the result -> recommmend cutting the losses
+        if (
+            experimentInsightType === InsightType.FUNNELS &&
+            funnelResultsPersonsTotal > Math.min(recommendedSampleSize, 500)
+        ) {
+            return (
+                <LemonBanner type="warning" className="mt-4">
+                    You've reached a robust sample size for your experiment, but the results are still inconclusive.
```

Comment:
nit: banner is the same, combine the trend & funnel conditions?

Reply:
first sentence is different 😬